WooCommerce/WooCommerceUITests/Mocks/mappings/jetpack-blogs/wc/payments/complete_cash_simple_payment.json

```diff
@@ -0,0 +1,115 @@
+{
+    "request": {
+        "method": "POST",
+        "urlPath": "/rest/v1.1/jetpack-blogs/161477129/rest-api/",
+        "bodyPatterns": [
+            {
+                "matches": ".*path=/wc/v3/orders/630%26_method%3Dpost"
+            },
+            {
+                "contains": "%7B%22status%22%3A%22completed"
+            }
```

Comment:
adding this note as a tip for future self or anyone looking to add a POST request - please check the body patterns and use encoded values for special characters so that the request can match, if not, the request might(will?) fail with a `<<<<< Body does not match` error. 

i spent too long trying to figure out what the issue was before finding out that it was the special characters...